eval/tools/curate-consistency-lemmas.sh

```
@@ -0,0 +1,69 @@
+    *ContradictoryAxioms*|*"status Theorem"*|*Unsatisfiable*) echo "VACUOUS $name" ;;
+    *) if [ -z "$eout" ] || [ -z "$vout" ]; then echo "NORESULT $name"; else echo "KEEP $name"; fi ;;
+  esac
+}
+export -f run_one
+export curate_tim
+
+# NUL-delimit the file list: some lemma names contain a single quote (e.g.
+# small_drinkers'_paradox), which the default whitespace/quote-processing xargs
+# rejects with "unmatched single quote".  -print0 | xargs -0 passes each path
+# verbatim; head -z keeps the optional limit NUL-aware.
+{ [ "$limit" -gt 0 ] && find "$work/problems" -name '*.p' -print0 | head -z -n "$limit" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)